requirements_pt.txt

```diff
@@ -1,4 +1,4 @@
-auto-round
+auto-round @ git+https://github.com/intel/auto-round.git@c7751c49853eb3497e7df9b92dc4733a80df34a0
```

Comment:
we can remove the commit, v0.2 has been released.

Reply:
removed https://github.com/intel/neural-compressor/pull/1810/commits/a8fbd53046b5bbf4c6f09425bdcd74dcab884fd7